packages/patternfly-3/patternfly-react-extensions/src/components/ClassificationBanner/ClassificationBanner.js

```diff
@@ -0,0 +1,131 @@
+
+    this.state = {
+      closed: props.closed || false
+    }
+
+  }
+
+  onClose = ()=>{
+    this.setState({closed:true})
+  }
+
+  componentWillReceiveProps(nextProps) {
+    
+    if (nextProps.closed !== this.state.closed) {
+      this.setState({ closed: nextProps.closed});
+    }
+  }
+
+  render(){
+    const { 
+      children, 
+      bottomBanner,
+      hostName,
+      userName,
+      closeButton,
+      userNamePosition,
+      hostNamePosition,
+      bannerColor,
+      title,
+      ...props } = this.props;
+
+      const defaultColors = {
+        "pf-red": 1,
+        "pf-blue": 1,
+        "pf-green": 1
+      }
+
+    const classificationBannerClasses = {
+      top: classNames('classification-banner-pf-banner',
+      'classification-banner-pf-banner-top',
+      {
+        'classification-banner-pf-hide': this.state.closed,
+      },
+      defaultColors[bannerColor]?('classification-banner-'+bannerColor):''),
+      bottom: classNames(
+      'classification-banner-pf-banner',
+      defaultColors[bannerColor]?('classification-banner-'+bannerColor):'',
+      {
+      'classification-banner-pf-hide':(!bottomBanner)||this.state.closed
+      },
+      'classification-banner-pf-banner-bottom'),
+      closeButton: classNames({
+        'classification-banner-pf-close pficon-error-circle-o': closeButton
+      }),
+      children: classNames({
+        'classification-banner-pf-children-no-bottom':(!this.state.closed && !bottomBanner),
+        'classification-banner-pf-children':(!this.state.closed && bottomBanner)})
+    }
+
+    var leftLabels=[hostNamePosition==='left'?(<span>{hostName}</span>):null,
+    userNamePosition==='left'?(<span>{userName}</span>):null],
+    rightLabels=[hostNamePosition==='right'?(<span>{hostName}</span>):null,
+    userNamePosition==='right'?(<span>{userName}</span>):null];
+
+    var bannerBackgroundStyle = defaultColors[bannerColor]?{}:{'background':bannerColor};
+    
+    return(<div>
+      <nav style={bannerBackgroundStyle} className={classificationBannerClasses.top} >
+        <div className={"classification-banner-pf-banner-left"}>
+          {leftLabels[0]}
+          {leftLabels[1]}
+        </div>
+        <div className={"classification-banner-pf-classification-level"}>{title}</div>
+        <div className={"classification-banner-pf-banner-right"}>
+        <i id="classification-banner-close-btn" className={classificationBannerClasses.closeButton} onClick={this.onClose}></i>
+          {rightLabels[0]}
+          {rightLabels[1]}
+        </div>
+
+      </nav>
+      <div className={classificationBannerClasses.children}>
+      {children}
+      </div>
+      <footer style={bannerBackgroundStyle} className={classificationBannerClasses.bottom}>
+        <div className={"classification-banner-pf-classification-level"}>{title}</div>
+      </footer>
+    </div>)
+
+  }
+}
+
+ClassificationBanner.propTypes = {
+
+  children: PropTypes.node,
+  hostName: PropTypes.string,
```

Comment:
propTypes need to be commented so they show in the storybook info doc.

Reply:
> propTypes need to be commented so they show in the storybook info doc.

@jeff-phillips-18 done